code/modules/vore/resizing/resize_vr.dm

```diff
@@ -106,15 +132,18 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
 
 
 /mob/living/proc/set_size()
-	set name = "Adjust Mass"
+	set name = "Adjust Size"
 	set category = "Abilities" //Seeing as prometheans have an IC reason to be changing mass.
 
-	var/nagmessage = "Adjust your mass to be a size between 25 to 200% (DO NOT ABUSE)"
```

Comment:
If theres' a "DO NOT ABUSE" clause in all capital letters, it stands to reason you should add it at the end of the edited message as well.